pom.xml

```diff
@@ -56,16 +56,16 @@
 
         <maven.compiler.parameters>true</maven.compiler.parameters>
 
-        <quarkus.platform.version>2.2.2.Final</quarkus.platform.version>
+        <quarkus.platform.version>2.6.2.Final</quarkus.platform.version>
```

Comment:
I think the upcoming product release is 2.7.5. Does that change anything with the other dependency version updates?

Reply:
> I think the upcoming product release is 2.7.5. Does that change anything with the other dependency version updates?

Yes, 2.7.5 is the intended final state.  We're waiting to hear back on support of java operator sdk 3.0.7 and fabric8 5.12.2

Reply:
See the next commit - switching to the 3.0.x sdk is a larger change that I haven't fully vetted.  There are some additional follow on items this does offer - such as access to the primary resource cache.